lib/scholar/linear/linear_regression.ex

```diff
@@ -61,7 +61,7 @@ defmodule Scholar.Linear.LinearRegression do
         >
       }
   """
-  deftransform fit(a, b, opts \\ []) do
+  deftransform fit(x, y, opts \\ []) do
```

Comment:
I think that for this specifically a and b is the more semantic choice, given we're generally talking about AX = B

Reply:
Perhaps mathematically, but from an API perspective I would say it's surprising and also not obvious to a beginner. Tbh I had to look up the order myself when it gave me hints in Livebook